WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -369,9 +369,12 @@ private extension StorePickerViewController {
             updateActionButtonAndTableState(animating: false, enabled: false)
             enterSiteAddressButton.isHidden = false
             newToWooButton.isHidden = false
-        default:
+        case .available(let sites):
             enterSiteAddressButton.isHidden = true
             newToWooButton.isHidden = true
+            if sites.allSatisfy({ $0.isWooCommerceActive == false }) {
+                updateActionButtonAndTableState(animating: false, enabled: false)
+            }
```

Comment:
forgot to mention, it might be nice moving this logic to the view model for unit testing 🙂

Reply:
Good idea! However it seems that `updateActionButtonAndTableState` is called in quite a few places, which means this would require a bigger refactor to move the old logic from the view controller to the view model. I'd say we do this in a separate task to be able to write more thorough tests.